mayastor-test/e2e/basic_volume_io/basic_volume_io_test.go

```diff
@@ -0,0 +1,73 @@
+// JIRA: CAS-505
+// JIRA: CAS-506
+package basic_volume_io_test
+
+import (
+	"e2e-basic/common"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var defTimeoutSecs = "90s"
+
+func TestBasicVolumeIO(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Basic volume IO tests, nvm-of tcp and iSCSI")
```

Comment:
Typo: NVMe, capitalisation should be consistent with message on line 53.

Reply:
Done